tefla/tune.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import argparse, json
+import tensorflow as tf
+from tefla.core.dir_dataset import DataSet
+from tefla.core.iter_ops import create_training_iters
+from tefla.core.learning import SupervisedLearner
+from tefla.da.standardizer import NoOpStandardizer
+from tefla.utils import util
+from tefla.core.hyperband import Hyperband
+from tefla.utils.hyperband_utils import mnist_dataset
```

Comment:
@pikkaay this shouldn't be dependent on Mnist dataset,  we should be able to pass the dataset from config file.